src/Helper/DbTrait.php

```diff
@@ -71,8 +71,8 @@ public function haveRecord($model, $data = [])
             $this->insertedRecords[$model] = [];
         }
 
-        $this->insertedRecords[$model][$table->primaryKey() . ' IN'][] = $data->{$table->primaryKey()};
-        return $data->{$table->primaryKey()};
+        $this->insertedRecords[$model][$table->getPrimaryKey() . ' IN'][] = $data->{$table->getPrimaryKey()};
+        return $data->{$table->getPrimaryKey()};
```

Comment:
getPrimaryKey() can return array, I hate the way it can `string|array`... 
Maybe we assign this to a var first and throw RuntimeException if array?